Source/Python/WarpXWrappers.cpp

```diff
@@ -652,6 +652,37 @@ namespace
         particle_buffers.clearParticles();
     }
 
+    void warpx_depositChargeDensity (const char* char_species_name, int lev) {
+        // this function is used to deposit a given species' charge density
+        // in the rho_fp multifab which can then be accessed from python via
+        // pywarpx.fields.RhoFPWrapper()
```

Comment:
Can you add this as doxygen string in the declaration in the header file, please? :)